content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,41 @@
+---
+linkTitle: Asset Processor Settings
+title: Asset Processor Settings
+description: Commandline settings for AssetProcessor and AssetBuilder
+weight: 900
+toc: true
+---
+
+The Asset Processor and Asset Processor Batch expose a number of commandline settings that can be used when manually launching.
+
+## General settings
+When manually launching Asset Processor or Asset Processor Batch you can set the following general properties:
+
+| Setting            | Description                                                         |
+|--------------------|---------------------------------------------------------------------|
+| project-name       | Name of the current project.                                        |
+| project-cache-path | Full path to the project cache folder.                              |
+| project-path       | Supplies the path to the project that the AP should use (required). | 
+| engine-path        | Supplies the engine path to the engine.                             |
+
+
+## Advanced Settings
+You can also control the behavior using the following command line options:
+
+| Setting                         | Description                                                                                                                                                                                                 | Notes                                    |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------|
+| waitOnLaunch                    | Briefly pauses Asset Processor during initialization. Useful if you want to attach a debugger.                                                                                                              | Defaults to false for AP, true for batch |
```

Comment:
The comment says this is false for AP and true for batch by default.  Is that true?  I don't think APB normally pauses to wait for a debugger

Reply:
Opps, wrong line. Am still auditing AP vs AP Batch changes. Will update.